### ISSUE TITLE
AttributeError when retrieving BoxPlayer in Espn API 0.5.0

### ISSUE THREAD

**Issue by zack-klein** [cwendt94/espn-api#110]
First off -- awesome package! I'm using it to help build some analytics into my league and it has been a lifesaver. Really helpful wrapper and very intuitive to use.

This morning, I noticed an error I haven't seen before with the release of `0.5.0`. When trying to grab box scores for a particular week, I get an `AttributeError` that `NoneType` has no attribute `get`. Looks like `self.stats` [here](https://github.com/cwendt94/espn-api/blob/master/espn_api/football/box_player.py#L26) is getting set as `None` somewhere. Interestingly enough, this only happens with some weeks (not all), so my assumption is some edge case in my league is causing this.  I can keep digging into what is special about that particular week and report back if I find anything. Reverting back to `0.4.2` also fixes this issue.

My full error output is:
```
>>> b = league.box_scores(week=6)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/zacharyjklein/.pyenv/versions/3.8.0/lib/python3.8/site-packages/espn_api/football/league.py", line 202, in box_scores
    box_data = [BoxScore(matchup, pro_schedule, positional_rankings, week) for matchup in schedule]
  File "/Users/zacharyjklein/.pyenv/versions/3.8.0/lib/python3.8/site-packages/espn_api/football/league.py", line 202, in <listcomp>
    box_data = [BoxScore(matchup, pro_schedule, positional_rankings, week) for matchup in schedule]
  File "/Users/zacharyjklein/.pyenv/versions/3.8.0/lib/python3.8/site-packages/espn_api/football/box_score.py", line 9, in __init__
    self.home_lineup = [BoxPlayer(player, pro_schedule, positional_rankings, week) for player in home_roster]
  File "/Users/zacharyjklein/.pyenv/versions/3.8.0/lib/python3.8/site-packages/espn_api/football/box_score.py", line 9, in <listcomp>
    self.home_lineup = [BoxPlayer(player, pro_schedule, positional_rankings, week) for player in home_roster]
  File "/Users/zacharyjklein/.pyenv/versions/3.8.0/lib/python3.8/site-packages/espn_api/football/box_player.py", line 27, in __init__
    self.points = stats.get('points', 0)
AttributeError: 'NoneType' object has no attribute 'get'
```

Let me know if I can provide any more info. Thanks so much!

**Comment by cwendt94** [comment]
Thanks for reporting this! I did a little debugging and found that it's happening to players on their bye week. I need to default the value to `{}` if week is not found.

I will have a fix up shortly!

**Comment by cwendt94** [comment]
Fixed in package 0.5.1! Let me know if you run into any other bugs!

**Comment by zack-klein** [comment]
Can confirm `0.5.1` fixes the issue - thanks so much!
